### PR TITLE
begin to support DME billing from misc_billing_options form

### PR DESCRIPTION
**Pull request by stephenwaite**
hopefully won't rock the billing boat ;) too bad

### CODE REVIEW

File: library/gen_x12_837.inc.php [hunk 1]
```diff
@@ -1076,15 +1078,16 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim = false)
...
         }
 
-    // Segment REF (Prescription or Compound Drug Association Number) omitted.
 
     // Loop 2420A, Rendering Provider (service-specific).
     // Used if the rendering provider for this service line is different
     // from that in loop 2310B.
-    //
-        if ($claim->providerNPI() != $claim->providerNPI($prockey)) {
+    // commented block below temporarily since this is removed in 5010
```
Comment (bradymiller) [inline library/gen_x12_837.inc.php]: If this is truly not needed for 5010 and no longer used, then prob best to completely remove it.

Comment (stephenwaite) [reply library/gen_x12_837.inc.php]: ok, will do, thanks

Comment (stephenwaite) [reply library/gen_x12_837.inc.php]: asked duc tran for current 5010 specs since he has a relationship or possibly even works for the X12 gang

**Comment by stephenwaite** [comment]
$$$ == 630$ on github :)

**Comment by bradymiller** [comment]
@stephen-smith ,
If this is testing well, let me know and I'll bring it into codebase (should this go into next patch which will release very soon or in a subsequent patch to allow some more testing?)

Also, as I recall, I remember somebody bringing up in past that do not need to pay for 5010 specs. Was that you @sunsetsystems ?

**Comment by bradymiller** [comment]
sorry @stephen-smith , I did it again (mentioned you by mistake), @stephenwaite , see my post above.

**Comment by stephenwaite** [comment]
thanks @bradymiller , it's testing well to the point where have verified the output looks compliant but that'd be nice to put in a patch for the user at chat.open-emr..org and can follow up on his claims to see if it's indeed valid

ps that sounds familiar about not paying for the specs

**Comment by bradymiller** [comment]
ok, gonna bring this in (but will bring it in on patch 3 rather than 2) to codebase now. Thanks for the improvement! -brady

**Pull request merged**